core/commands/VersionGetCommand.php

```diff
@@ -100,9 +121,9 @@ protected function process() {
 		if( is_null( $this->version_id ) ) {
 			$t_versions = version_get_all_rows(
 				$this->project_id,
-				/* released */ $this->query( 'released' ),
-				/* obsolete */ $this->query( 'obsolete', false ),
-				/* inherit */ $this->query( 'inherit', false ) );
+				/* released */ $this->query( 'released', $r_released ),
+				/* obsolete */ $this->query( 'obsolete', $r_obsolete ),
+				/* inherit */ $this->query( 'inherit', $r_inherited ) );
```

Comment:
Command::query()'s 2nd parameter is supposed to be the default value, which would have been set in rest_project_version_get() when retrieving the request's parameters so this change is probably not necessary.